src/assets/scss/components/seriesTable.scss

```diff
@@ -11,3 +11,8 @@
     color: var(--pf-global--danger-color--100);
     margin-left: 0.5em;
 }
+
+.results-spinner-container {
+    display: flex;
+    justify-content: center;
+}
```

Comment:
@janakitti can you do me a favor and refactor this file to have 2 space indentation so it's consistent with the rest of the codebase.